packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -1918,6 +1936,147 @@ function updateDehydratedSuspenseComponent(
+  // Mark any Suspense boundaries with fallbacks as having work to do.
+  // If they were previously forced into fallbacks, they may now be able
+  // to unblock.
+  let node = firstChild;
+  while (node !== null) {
+    if (node.tag === SuspenseComponent) {
+      const state: SuspenseState | null = node.memoizedState;
+      if (state !== null) {
+        if (node.expirationTime < renderExpirationTime) {
+          node.expirationTime = renderExpirationTime;
+        }
+        let alternate = node.alternate;
+        if (
+          alternate !== null &&
+          alternate.expirationTime < renderExpirationTime
+        ) {
+          alternate.expirationTime = renderExpirationTime;
+        }
+        scheduleWorkOnParentPath(node.return, renderExpirationTime);
+      }
+    } else if (node.child !== null) {
+      node.child.return = node;
+      node = node.child;
+      continue;
+    }
+    if (node === workInProgress) {
+      return;
+    }
+    while (node.sibling === null) {
+      if (node.return === null || node.return === workInProgress) {
+        return;
+      }
+      node = node.return;
+    }
+    node.sibling.return = node.return;
+    node = node.sibling;
+  }
+}
+
+type SuspenseListRevealOrder = 'together' | void;
+
+function updateSuspenseListComponent(
+  current: Fiber | null,
+  workInProgress: Fiber,
+  renderExpirationTime: ExpirationTime,
+) {
+  const nextProps = workInProgress.pendingProps;
+  const revealOrder: SuspenseListRevealOrder = nextProps.revealOrder;
+  const nextChildren = nextProps.children;
+
+  let nextChildFibers;
+  if (current === null) {
+    nextChildFibers = mountChildFibers(
+      workInProgress,
+      null,
+      nextChildren,
+      renderExpirationTime,
+    );
+  } else {
+    nextChildFibers = reconcileChildFibers(
+      workInProgress,
+      current.child,
+      nextChildren,
+      renderExpirationTime,
+    );
+  }
+
+  let suspenseContext: SuspenseContext = suspenseStackCursor.current;
+
+  let shouldForceFallback = hasSuspenseContext(
+    suspenseContext,
+    (ForceSuspenseFallback: SuspenseContext),
+  );
+
+  if ((workInProgress.effectTag & DidCapture) !== NoEffect) {
+    // This is the second pass. In this pass, we should force the
+    // fallbacks in place.
+    shouldForceFallback = true;
+  }
+
+  if (shouldForceFallback) {
+    suspenseContext = setShallowSuspenseContext(
+      suspenseContext,
+      ForceSuspenseFallback,
+    );
+    workInProgress.memoizedState = true;
+  } else {
+    let didForceFallback = current !== null && current.memoizedState === true;
+    if (didForceFallback) {
+      // If we previously forced a fallback, we need to schedule work
+      // on any nested boundaries to let them know to try to render
+      // again. This is the same as context updating.
+      propagateSuspenseContextChange(
+        workInProgress,
+        nextChildFibers,
+        renderExpirationTime,
+      );
+    }
+    workInProgress.memoizedState = false;
```

Comment:
Is this going to mess with hidden class?

Reply:
Is this a new comment? What happened to the other one that I already answered?

Reply:
Lol. I thought I hadn't send it because I couldn't find it today either (and didn't see your reply to it).

Reply:
I made it an object since I will do that in the next PR anyway.